build/GenerateCppJsRulesDescription.ps1

```diff
@@ -0,0 +1,65 @@
+param
+(
+    [Parameter(Mandatory = $true, HelpMessage = "target path to store the rules description")]
+    [ValidateScript({Test-Path $_})]
+    [string]
+    $targetFolder,
+
+    [Parameter(Mandatory = $true, HelpMessage = "path to rule extractor (from sonarlint-core). For example 'c:\src\sonarlint-rule-extractor-2.18-SNAPSHOT-jar-with-dependencies.jar' ")]
```

Comment:
@Evangelink you probably already know, but shift + scroll of mouse helps a little bit